packages/command-palette/src/commands.tsx

```diff
@@ -1363,6 +1363,43 @@ export function useCommands() {
 			manageSettingsGeneral: {
 				name: 'manageSettingsGeneral',
 				label: __( 'Manage general settings', __i18n_text_domain__ ),
+				searchLabel: [
+					_x(
+						'change site title',
+						'Keyword for the Manage general settings command',
+						__i18n_text_domain__
+					),
+					_x(
+						'change tagline',
+						'Keyword for the Manage general settings command',
+						__i18n_text_domain__
+					),
+					_x(
+						'change site icon',
```

Comment:
Wdyt about adding `change icon` as well?

Reply:
I wanted to use the same wording we use in the settings page:

<img width="742" alt="Screenshot 2024-06-03 at 12 25 38" src="https://github.com/Automattic/wp-calypso/assets/1233880/778cc921-8ea2-4d8f-a902-31fb310ce29c">